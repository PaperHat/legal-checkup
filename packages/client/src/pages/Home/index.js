import * as React from 'react'
import { Helmet } from 'react-helmet'

// eslint-disable-next-line
import Footer from '../../components/layout/Footer'
import Header from '../../components/layout/Header'
import HomeComponent from '../../components/home'

import Desktop from '../../components/responsive/Desktop'
import Mobile from '../../components/responsive/Mobile'
import Tablet from '../../components/responsive/Tablet'
import { DESKTOP, MOBILE, TABLET, WEBSITE } from '../../constants'

function getLayout (format) {
  return (
    <>
      <Header format={format} />
      <HomeComponent format={format} />
      <Footer format={format} />
    </>
  )
}

export default function Home () {
  return (
    <>
      <Helmet>
        <title>{WEBSITE}</title>
      </Helmet>
      <Mobile>{getLayout(MOBILE)}</Mobile>
      <Tablet>{getLayout(TABLET)}</Tablet>
      <Desktop>{getLayout(DESKTOP)}</Desktop>
    </>
  )
}
