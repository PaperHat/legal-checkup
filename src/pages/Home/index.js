import React from 'react'

import { Layout } from '../../components'

export default function Home () {
  return (
    <Layout>
      <p>This is the Home page</p>
    </Layout>
  )
}
