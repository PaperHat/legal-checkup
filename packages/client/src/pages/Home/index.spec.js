import React from 'react'
import { shallow } from 'enzyme'

import Home from './'

describe('components:Home', () => {
  it('renders the Home page', () => {
    expect(toJson(shallow(<Home />).dive())).toMatchSnapshot()
  })
})
