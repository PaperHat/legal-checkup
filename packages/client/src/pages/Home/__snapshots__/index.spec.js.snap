// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages:Home renders the Home page 1`] = `null`;
